docs/topics/api/addons.rst

```diff
@@ -358,3 +360,25 @@ This endpoint allows you to fetch an add-on EULA and privacy policy.
 
     :>json string|object|null eula: The text of the EULA, if present (See :ref:`translated fields <api-overview-translations>`).
     :>json string|object|null privacy_policy: The text of the Privacy Policy, if present (See :ref:`translated fields <api-overview-translations>`).
+
+
+--------------
+Language Tools
+--------------
+
+.. _addon-language-tools:
+
+This endpoint allows you to list all public language tools add-ons available
+on AMO.
+
+.. http:get:: /api/v3/addons/language-tools/
+
+    .. note::
+        Because this endpoint is intended to be used to feed a page that
+        displays all available language tools in a single page, it is not
+        paginated as normal, and instead will return all results without
+        obeying regular pagination parameters.
+
+    :query string app: Mandatory. Filter by :ref:`add-on application <addon-detail-application>` availability.
+    :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
+    :>json array results: An array of :ref:`add-ons <addon-detail-object>`, but with only the following fields present: ``id``, ``current_version``, ``default_locale``, ``locale_disambiguation``, ``name``, ``target_locale``, ``type`` and ``url``.
```

Comment:
So, I wanted to do that... but sphinx-httpcontrib is terrible. It does not let you use `:>json` in arbitrary contexts (needs to follow a `<verb>::<path>` section), and I can't use the same workaround we have for the add-on object, because we don't have a detail page. 

I tried to cheat and mimic the styling but it looked goofy because the font-size and margin were different.

So this was the simplest hack I could think of. The proper solution is probably going to be having a separate section for each type of ressource we have, and document the fields there, without using `:>json` at all.